vault/logical_system_quotas.go

```diff
@@ -20,6 +20,10 @@ func (b *SystemBackend) quotasPaths() []*framework.Path {
 					Type:        framework.TypeBool,
 					Description: "If set, starts audit logging of requests that get rejected due to rate limit quota rule violations.",
 				},
+				"enable_response_headers": {
```

Comment:
Maybe this is sufficient, but I wonder if we need type specific response header knobs. For example, `enable_rate_limit_response_headers` and `enable_lease_count_response_headers` and so on.

@briankassouf thoughts?

Reply:
That is an option, although I can't think of strong reasons why we'd want that, but perhaps flexibility triumphs simplicity?

Reply:
The reason I wondered about having separate knobs for these was that the security implication of revealing the rate limiting information and the lease count information could be very different. I am guessing that some folks may be okay to reveal the rate information but not of leases. WDYT?

Reply:
It can't hurt I suppose. It really depends on how many types of resource quotas we plan on supporting in the future. Perhaps we support separate knobs for now. Any objections @briankassouf?

Reply:
I'm going to go ahead and apply your suggestion @vishalnayak 👍